### PR TITLE
WildCards Import add documentation

### PR DESCRIPTION
**Pull request by kentr0w**
### What's done:
    Added info

### CODE REVIEW

File: diktat-rules/src/main/resources/diktat-analysis.yml [hunk 1]
```diff
@@ -148,7 +148,8 @@
   configuration: {}
 - name: FILE_WILDCARD_IMPORTS
```
Comment (akuleshov7) [inline diktat-rules/src/main/resources/diktat-analysis.yml:149]: let's start writing documentation here as comments to yml file

Comment (akuleshov7) [reply diktat-rules/src/main/resources/diktat-analysis.yml:149]: @petertrr @kentr0w @aktsay6

**Review by akuleshov7 — CHANGES_REQUESTED** [review]
Denis, be quick! Fix it and merge

**Review by petertrr — APPROVED** [review]

**Review by aktsay6 — APPROVED** [review]
lgtm

**Pull request merged**